guides/common/modules/ref_virt-who-configuration-overview.adoc

```diff
@@ -0,0 +1,27 @@
+[id="virt-who-configuration-overview"]
+= virt-who configuration overview
+
+You configure virt-who by creating a virt-who configuration on {ProjectServer}.
+Then, you deploy the configuration on a target host such as a hypervisor or a dedicated {el} server.
+
+.Preparing your environment
+* Import a subscription manifest that includes a host-based subscription into {ProjectServer}.
+* Register the target host to {ProjectServer}.
+* Create a read-only user for virt-who on the target host.
+
+.Procedure
+. Create a virt-who configuration on {ProjectServer}.
+. Deploy the virt-who configuration on the target host.
```

Comment:
question (non-blocking): is there a way to link to the relevant procedures? If so, that'd improve the UX.

Reply:
We have separate assemblies for each virt platform, so it is not possible to link directly from a generic procedure.